.test-infra/metrics/grafana/dashboards/perftests_metrics/SideInput_Load_Tests.json

```diff
@@ -21,7 +21,7 @@
   "links": [],
   "panels": [
     {
-      "content": "The following options are common to all tests:\n* key size: 100B\n* value size: 900B\n* number of workers: 10\n* size of the window (if fixed windows are used): 1 second\n\nAdditional common options for Dataflow:\n* experiments: use_runner_v2\n* autoscaling_algorithm: NONE\n\n\n[Jenkins job definition (Python, Dataflow)](https://github.com/apache/beam/blob/master/.test-infra/jenkins/job_LoadTests_SideInput_Python.groovy)\n\n",
+      "content": "The following options are common to all tests:\n* key size: 100B\n* value size: 900B\n* number of workers: 10\n* size of the window (if fixed windows are used): 1 second\n\nAdditional common options for Dataflow:\n* experiments: use_runner_v2\n* autoscaling_algorithm: NONE\n\n\n[Jenkins job definition (Python, Dataflow)](https://github.com/apache/beam/blob/master/.test-infra/jenkins/job_LoadTests_SideInput_Python.groovy)\n\nUntil the issue [BEAM-11427](https://issues.apache.org/jira/browse/BEAM-11427) in Go SDK is resolved, sideinput iteration test have 400MB, instead of 10GB.",
```

Comment:
The comment is OK, but you should also specify additional options for Flink. And while there is a link to Python Jenkins job, you could also add links to your newly defined jobs.